### PR TITLE
Update run.md

### PR DESCRIPTION
**Pull request by kurianbenoy**
closes https://github.com/iterative/dvc/issues/6186  Documentation changes.

Thanks 🙏🏾

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Please only state which one issue this PR closes in the description so they are linked. Thanks

**Comment by jorgeorpinel** [comment]
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: content/docs/command-reference/exp/run.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # exp run
 
-Run an [experiment](/doc/command-reference/exp): reproduce a variation of a
-committed [pipeline](/doc/command-reference/dag) in a hidden project branch.
+Run or resume an [experiment](/doc/command-reference/exp).
 
 > Similar to `dvc repro` but for
 > [experimentation](/doc/user-guide/experiment-management).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/exp/run.md:6]: Removing per https://github.com/iterative/dvc/pull/6197#issuecomment-870230101
```suggestion
```

**Review by jorgeorpinel — APPROVED** [review]
Just one logistical ask so we can merge this ☝️

**Comment by kurianbenoy** [comment]
Everything LGTM @jorgeorpinel . Thanks

**Comment by jorgeorpinel** [comment]
Thanks @kurianbenoy ! Please link the PR to the issue it will close. Please see the link in my previous message.

**Comment by kurianbenoy** [comment]
I have linked the PR to the issue already it closes in DVC library.

**Comment by jorgeorpinel** [comment]
Right @kurianbenoy but you need to use a keyword to actually link them. Or use use the link button. Again, please see the article I provided. Thanks

**Comment by kurianbenoy** [comment]
Updated the description @jorgeorpinel . I didn't know github had such an awesome docs 👍🏾

**Comment by jorgeorpinel** [comment]
Thanks!

**Pull request merged**